tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +205,44 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin'] = function() {
+
+		var errorLogStub = sinon.stub( console, 'error' );
+
+		var onErrorStub = function() {
```

Comment:
```suggestion
		var runtimeErrorListener = function() {
```

This is not a stub but listener so should be named accordingly.